observability/metrics/src/main/scala/com/daml/metrics/InstrumentedExecutorServiceMetrics.scala

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.metrics
+
+import java.util
+import java.util.concurrent.{Callable, ExecutorService, Future, TimeUnit}
+
+import com.daml.metrics.api.MetricDoc.MetricQualification.Debug
+import com.daml.metrics.api.MetricHandle.{Counter, Meter, MetricsFactory, Timer}
+import com.daml.metrics.api.{MetricDoc, MetricsContext}
+
+import scala.jdk.CollectionConverters.{CollectionHasAsScala, SeqHasAsJava}
+
+class InstrumentedExecutorServiceMetrics(factory: MetricsFactory) {
+
+  @MetricDoc.Tag(
+    summary = "The number of tasks submitted to an instrumented executor.",
+    description = "Number of tasks that were submitted to the executor.",
+    qualification = Debug,
+    labelsWithDescription = Map(
+      "name" -> "The name of the executor service.",
+      "type" -> "The type of the executor service. Can be `fork_join` or `thread_pool`.",
```

Comment:
Suggestion to make the documentation more condensed:
```suggestion
      "type" -> "The type of the executor service: `fork_join` or `thread_pool`.",
```